sympy/integrals/manualintegrate.py

```diff
@@ -725,10 +725,18 @@ def trig_cotcsc_rule(integral):
                 cotcsc_csceven_condition: cotcsc_csceven
             })((a, b, m, n, integrand, symbol))
 
+def trig_sindouble_rule(integral):
+    integrand, symbol = integral
+    a = sympy.Wild('a', exclude=[1/(sympy.sin(2*symbol))])
```

Comment:
I see. I haven't seen exclude used like this, but I think it should work. I don't think the `1/` part is needed, though.